packages/website/sidebars.json

```diff
@@ -9,6 +9,7 @@
       "guides/integrating-the-feature-hub",
       "guides/writing-a-feature-app",
       "guides/writing-a-feature-service",
+      "guides/server-side-rendering-using-react",
```

Comment:
What do you think about moving this down at least one entry? I think sharing the browser history is more essential than ssr.